packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/setPayees.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  CONTRACT_TYPES,
+  ExecuteCommandConfig,
+  ExecutionContext,
+  makeExecuteCommand,
+  getRDD,
+} from '@chainlink/starknet-gauntlet'
+import { ocr2ContractLoader } from '../../lib/contracts'
+
+type PayeeConfig = {
+  transmitter: string
+  payee: string
+}
+
+type UserInput = PayeeConfig[]
+
+type ContractInput = [payees: PayeeConfig[]]
+
+const makeUserInput = async (flags, args): Promise<UserInput> => {
+  if (flags.input) return flags.input as UserInput
+  const rdd = getRDD(flags.rdd)
```

Comment:
this implementation means input must be provided directly through the `input` flag or through RDD

Reply:
Since this is an ExecuteCommand, `makeContractInput`'s `ctx.rdd` will be available and you can avoid loading this manually in `makeUserInput`.

then this becomes a two-liner, like `setBilling`:

```js
if (ctx.rdd) {
  const contract = ctx.rdd[CONTRACT_TYPES.AGGREGATOR][ctx.contractAddress];
  input = aggregator.payees
}
```

Reply:
thanks, changed to this implementation and added a manual way to define input params from cli args 👍